snapshots/artificial_intelligence/2024-02-15/epoch_llms.csv.dvc

```diff
@@ -0,0 +1,21 @@
+meta:
+  origin:
+    title: Large Language Model Performance and Compute
+    description: Epoch dataset on how performance on a MMLU language benchmark scales
+      with computational resources.
+    producer: David Owen (Epoch)
```

Comment:
Should it better be Epoch? as in [the charts](https://ourworldindata.org/grapher/ai-training-computation-vs-parameters-by-domain)

Reply:
hmm yeah I wasn't sure about this - the author provided the data to us directly so I wasn't sure but agree Epoch is probably better!